R/with.R

```diff
@@ -0,0 +1,125 @@
+#' Modify lintr defaults
```

Comment:
really wish github made it clearer whether this is just copy-pasted from the old file :|

Reply:
Yeah, detecting file splits in addition to file renames would be nice.